hm-basic-webapp/values.yaml

```diff
@@ -21,8 +21,9 @@ ingress:
   acmeRegistrationEmail: nobody@localhost.local
   path: /
   tlsSecretName: dev-localhost-local-tls
+  ingressClassName: nginx
   annotations:
-    kubernetes.io/ingress.class: nginx
+#    kubernetes.io/foo.class: bar
```

Comment:
we can also set the example like this
```
    annotations: {}
```

Reply:
I know, I left it like this so the comment helps the reader to know how to use it.

I can change it to `{}` if you like though.